roomserver/query/query.go

```diff
@@ -617,42 +617,42 @@ func (r *RoomserverQueryAPI) QueryStateAndAuthChain(
 func getAuthChain(
 	ctx context.Context, dB RoomserverQueryAPIEventDB, authEventIDs []string,
 ) ([]gomatrixserverlib.Event, error) {
-	var authEvents []gomatrixserverlib.Event
-
-	// List of event ids to fetch. These will be added to the result and
-	// their auth events will be fetched (if they haven't been previously)
 	eventsToFetch := authEventIDs
```

Comment:
Can you please keep that comment and add a mention of where we fetch events from (i.e. the DB afaiu)? It wasn't straightforward to me where the events were being fetched from (DB? another server?) and more doc never hurts.

Reply:
I've updated the comments to be a bit clearer as to what's going on :-)